charmcraft/application/commands/init.py

```diff
@@ -76,7 +76,7 @@
         A basic Kubernetes charm for a 12-factor Go app.
 
 Depending on the profile choice, Charmcraft will setup the following tree of
-files and directories:
+files and directories::
```

Comment:
This is going to make the colon appear twice on the CLI.

Reply:
Per external discussion:

- Short term: We're going to accept this for now since it's less bad than how the sphinx docs look.
- Medium term: https://github.com/canonical/craft-cli/issues/351
- Long term: craft-cli is going to rethink how this is done.